@kiva/kv-tokens/primitives.json

```diff
@@ -163,6 +163,15 @@
 				"background": {
 					"primary": "#454545"
 				}
+			},
+			"dark-stone": {
+				"text": {
+					"primary": "#F5F5F5",
+					"action": "#F5F5F5"
```

Comment:
Shouldn't this be `#F3F1EF` ? That is the lightest Stone theme color. In some of the ecosystem design examples the lightest color in the theme is used for the text color when over the dark color as a bg.

Reply:
https://www.figma.com/design/ns8HGt1yjQBfiC1OKlHU40/Legacy---%F0%9F%8C%BF-Ecosystem---Kiva-Brand-2022?node-id=3077-2580&t=ZAQmb3Azw5Bd20Mm-0

Reply:
We may also be able to use some values from the new "Default" dark theme that's in progress to guide some of the other variables. In particular, setting text-action to the same value means links would be indistinguishable...
https://www.figma.com/design/ns8HGt1yjQBfiC1OKlHU40/Legacy---%F0%9F%8C%BF-Ecosystem---Kiva-Brand-2022?node-id=4315-2579&t=ZAQmb3Azw5Bd20Mm-0

Reply:
That's a nice call but we have `#F5F5F5` in figma, said that, should this be a new theme different than Stone?
Or maybe other call to design to follow our current themes?

Reply:
the figma design I see uses stone-4 and marigold. I don't think it makes sense to name this theme based on either of those.